### PR TITLE
Add compat data for transform CSS properties

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds compat data for the following CSS properties:

* [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) (Note: there are many value features that need to be added here; they'll come along in future PRs.)
* [`transform-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-box)
* [`transform-origin`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin)
* [`transform-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-style)

### CODE REVIEW

File: css/properties/transform-box.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                "version_added": "43",
+                "flag": {
+                  "type": "preference",
+                  "name": "svg.transform-box.enabled",
+                  "value_to_set": "true"
+                }
+              },
+              {
+                "version_added": "41",
+                "version_removed": true,
```
Comment (teoli2003) [inline css/properties/transform-box.json]: The removal happened in "43". See https://bugzilla.mozilla.org/show_bug.cgi?id=1175492 .

File: css/properties/transform-box.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "43"

File: css/properties/transform-box.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                  "name": "svg.transform-origin.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "false"

File: css/properties/transform-box.json [hunk 4]
```diff
@@ -0,0 +1,95 @@
+{
+  "css": {
+    "properties": {
+      "transform-box": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/transform-box",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "64" ( https://www.chromestatus.com/feature/6643927633362944 )

File: css/properties/transform-box.json [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "64" ( https://www.chromestatus.com/feature/6643927633362944 )

File: css/properties/transform-box.json [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+            },
+            "edge": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "false"

File: css/properties/transform-box.json [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+            },
+            "edge_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "false"

File: css/properties/transform-box.json [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "51" ( https://www.chromestatus.com/feature/6643927633362944 )

File: css/properties/transform-box.json [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "51" ( https://www.chromestatus.com/feature/6643927633362944 )

File: css/properties/transform-box.json [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+    "properties": {
+      "transform-box": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/transform-box",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/transform-box.json]: "64" ( https://www.chromestatus.com/feature/6643927633362944 )

File: css/properties/transform-origin.json [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+                "prefix": "-webkit-",
+                "version_added": "44",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": {
```
Comment (teoli2003) [inline css/properties/transform-origin.json]: Same as "firefox"

File: css/properties/transform.json [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+                "prefix": "-webkit-",
+                "version_added": "44",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": {
```
Comment (teoli2003) [inline css/properties/transform.json]: Same as "firefox"

**Comment by ddbeck** [comment]
OK, I've updated based on your comments, @teoli2003. Some changes required updates to the Opera browser data, though I'm not sure I did that perfectly (see cf8867b for details). Thanks for reviewing this PR!

**Pull request merged**